src/dynamic_programming/coin_change.jl

```diff
@@ -0,0 +1,66 @@
+"""
+    coin_change(coins::Vector{Int}, amount::Int)
+
+Given a vector `coins` of coin values, calculates the minimum number of coins that sums to `amount`.
+It's considered that a unlimited number of coins for each value is available.
+
+# Arguments:
+- `coins`: the coins values available
+- `amount`: the total amount that need to be summed to
+
+# Examples
+```jldoctest
+julia> n_coins, coins = coin_change([1, 3, 4, 7], 13)
```

Comment:
Documenter error:

```
 │ Subexpression:
│ 
│ n_coins, coins = coin_change([2, 4, 6], 23)
│ 
│ Evaluated output:
│ 
│ (-1, Int64[])
│ 
│ Expected output:
│  
```

Your expected output should be empty. If you want that, adding a semicolon should silence output:
```suggestion
julia> n_coins, coins = coin_change([1, 3, 4, 7], 13);
```